plugin.go

```diff
@@ -38,7 +38,7 @@ func (p Plugin) Exec() error {
 		}
 	}
 
-	err := writeNetrc(p.Netrc.Machine, p.Netrc.Login, p.Netrc.Password)
```

Comment:
@6543 Should we exit with an error in case `PLUGIN_HOME` is not set / empty.

Reply:
maybe we can set config home if empty with xdg home like in setHome?
https://github.com/woodpecker-ci/plugin-git/blob/3b7301e692d8fc8c85328d6c213e0fc2abeb2b8a/plugin.go#L332